app/helpers/promos_helper.rb

```diff
@@ -51,6 +51,25 @@ def publisher_current_referral_totals(publisher)
     }
   end
 
+  def channel_referral_totals(channel)
```

Comment:
I believe this is a duplicate of `aggregate_stats` on the `promo_registrations` model.

Reply:
Fixed! https://github.com/brave-intl/publishers/pull/2094/commits/c2ca42aeb6d72513d0167f2d56d7651973e8e470